src/main/java/uk/gov/laa/ccms/caab/constants/SessionConstants.java

```diff
@@ -112,6 +112,23 @@ public class SessionConstants {
    */
   public static final String APPLICATION_CLIENT_NAMES = "applicationClientNames";
 
+  /**
+   * Session attribute used to keep track of criteria when searching for an organisation.
+   */
+  public static final String ORGANISATION_SEARCH_CRITERIA = "orgSearchCriteria";
```

Comment:
do we want full names here - organisationSearchCriteria

Reply:
amended